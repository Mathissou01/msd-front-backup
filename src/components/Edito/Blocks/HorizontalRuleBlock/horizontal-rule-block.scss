@use "src/styles/01-settings/settings.colors" as colors;
@use "src/styles/02-tools/tools.include-media" as im;

.c-HorizontalRuleBlock {
  width: 100%;
  height: 0;
  margin: 16px 0;
  border: 1px solid colors.$blue-1;

  @include im.media(">=tablet") {
    margin: 28px 0;
    border: 1px solid colors.$gray-500;
  }
}
