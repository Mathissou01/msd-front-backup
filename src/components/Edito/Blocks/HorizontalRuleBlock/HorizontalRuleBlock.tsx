import React from "react";
import "./horizontal-rule-block.scss";

export default function HorizontalRuleBlock() {
  return <div className="c-HorizontalRuleBlock" />;
}
