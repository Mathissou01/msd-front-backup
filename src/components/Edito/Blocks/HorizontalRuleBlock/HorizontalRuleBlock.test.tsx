import { render } from "@testing-library/react";
import HorizontalRuleBlock from "./HorizontalRuleBlock";

describe("HorizontalRuleBlock", () => {
  it("renders", () => {
    const { container } = render(<HorizontalRuleBlock />);
    expect(container).toMatchSnapshot();
  });
});
