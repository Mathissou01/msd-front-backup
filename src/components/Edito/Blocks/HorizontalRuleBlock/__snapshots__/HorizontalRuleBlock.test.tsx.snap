// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HorizontalRuleBlock renders 1`] = `
<div>
  <div
    class="c-HorizontalRuleBlock"
  />
</div>
`;
